flow/project.py

```diff
@@ -2371,10 +2659,12 @@ def completed_operations(self, job):
 
     def _job_operations(self, job, only_eligible):
         "Yield instances of JobOperation constructed for specific jobs."
-        for name, op in self.operations.items():
-            if only_eligible and not op.eligible(job):
+        for name in self.operations.keys():
```

Comment:
This seems to fix the issue on Py35, although I'm a bit confused that this was not a problem earlier:
```suggestion
        for name in sorted(self.operations):
```